html/rules.html

```diff
@@ -1,75 +1,75 @@
-<p>6. Faction Metagaming. Do not move to the enemy factions base until at least 40 minutes into the round. Claiming to have &lsquo;found it&rsquo; is a bad excuse, and isn&rsquo;t going to work.</p>
-<p>7. Bad Deathclaw has a focus towards roleplay. Roleplay a believable character that could exist in the Fallout universe. Shitty memes and low roleplay gimmicks will not end well.</p>
-<p>8. Do not be unnecessarily hostile. Arguments are fine, being an asshole for no reason isn't. This includes taking it to OOC chat and making trouble.</p>
-<p>9. Do not attack or harm AFK/SSD players. You are free to take their items, but not attack them.</p>
-<p>10. No rule lawyering.</p>
+<p>0. Don’t be a Dick. If you are intentionally ruining another players experience or going out of your way to negatively impact the round, you can be punished for it..</p>
+<p>    
+	<br /><strong>General Rules</strong></p>
+<p>1. Admin Rulings are final. If you wish to contest a ruling, please go to the discord. In each situation, an admins ruling is how the rules are to be treated in that given situation.</p>
+<p>2. Don’t use OOC information IC. Don’t leak IC information to OOC.</p>
+<p>3. Play as the role you spawn in as, joining as a role to desert or otherwise undermine your faction is bannable.</p>
+<p>4. Bad Deathclaw is an RP server, and roleplay is expected. Failing to do so is punishable..</p>
+<p>4. Do not attack, assault or harm SSD/AFK players. You may take their equipment, but dragging them off or murdering them is not okay. Catatonic Players are free game.</p>
+<p>5. No Rules Lawyering.</p>
+<p>6. Name your character appropriately. Inappropriate names may result in a ban. Raider names may be ridiculous, but should not reference OOC memes or pop culture..</p>
 <p>
-    <br /><strong>B. Approach, Motivation and Escalation</strong></p>
-<p>1. Do not attack someone wordlessly.</p>
-<p>2. Establish hostility before attacking. This means using your words. Wait for them to react, whether words, brandishing a weapon, violence, or compliance.</p>
-<p>3. Escalation comes with reasoning and is mostly limited by role background. You shouldn&rsquo;t see Wastelanders and Settlers being mass-murderers, but Raiders are more likely.</p>
-<p>4. Fleeing confrontation constitutes escalation.</p>
-<p>5. If you suspect the other player is going to fail to escalate properly, it&rsquo;s better to let it pan out, then shoot first and have -you- look like the villain.</p>
+    <br /><strong>Escalation Rules</strong></p>
+<p>1. Legion and NCR do not need to escalate on one another, in any way. They may attack each other for any reason at any time.</p>
+<p>2. No other parties may enter into combat with Legion or NCR without being in recognisable faction uniform, of either Legion or NCR. Armor and Helmet count appropriately.</p>

```

Comment:
Decapitalize "helmet." Alter wording to "A recognisable faction uniform must include their armor and helmet."